andela/ah-fulldeck-frontend
# #163858430 user should be able to view and edit their profile

#### What does this PR do?
* Implement user view and edit their profile

#### Description of Task to be completed?
* A user can view their profile
* A user can edit their profile
#### How should this be manually tested?
* Clone the github repository `git clone https://github.com/andela/ah-fulldeck-frontend` on your local machine.
* Change your working directory to the project folder `cd ah-fulldeck-frontend`
* Checkout to thebranch `ft-user-profile-163858430` using the command `git checkout ft-user-profile-163858430`
* Install the required dependencies using `npm install`
* Run the local development server using `npm start` and you should get the following message:
* Go to the link `/profile/<username>` where the `username` is your registered username

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#162948946](https://www.pivotaltracker.com/story/show/163858430)

#### Screenshots (if appropriate)
* User view their profile
<img width="1171" alt="screenshot 2019-02-13 at 16 42 24" src="https://user-images.githubusercontent.com/32098817/52722051-f3f7bd80-2fbb-11e9-82b0-a90609040075.png">
*  User edit their profile
<img width="1186" alt="screenshot 2019-02-13 at 16 41 33" src="https://user-images.githubusercontent.com/32098817/52722121-0ffb5f00-2fbc-11e9-8d99-f4d9c6fbb4e6.png">
* Edited profile
<img width="1160" alt="screenshot 2019-02-13 at 18 35 57" src="https://user-images.githubusercontent.com/32098817/52723286-45a14780-2fbe-11e9-8440-b418cf2fe957.png">
